samples/api-client/mqtt_example/README.rst

```diff
@@ -0,0 +1,151 @@
+.. This file is automatically generated. Do not edit this file directly.
+
+Google Cloud IoT Core Python Samples
+===============================================================================
+
+.. image:: https://gstatic.com/cloudssh/images/open-btn.png
+   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=iot/api-client/mqtt_example/README.rst
+
+
+This directory contains samples for Google Cloud IoT Core. `Google Cloud IoT Core`_ allows developers to easily integrate Publish and Subscribe functionality with devices and programmatically manage device authorization.
+This example connects a device to `Google Cloud IoT Core`_ via MQTT, using a JWT for authentication. After connecting, the device publishes messages to the device's  MQTT topic at a rate of one per second.
+Note that before you can run this sample, you  must register a device as described  in the parent README. For the gateway samples, you must register and bind a device  as described in the  `Cloud IoT gateway docs <https://cloud.google.com/iot/docs/how-tos/gateways/#setup>`_.
+
+
+
+
+.. _Google Cloud IoT Core: https://cloud.google.com/iot/docs
+
+
+
+
+
+Setup
+-------------------------------------------------------------------------------
+
+
+Authentication
+++++++++++++++
+
+This sample requires you to have authentication setup. Refer to the
+`Authentication Getting Started Guide`_ for instructions on setting up
+credentials for applications.
+
+.. _Authentication Getting Started Guide:
+    https://cloud.google.com/docs/authentication/getting-started
+
+Install Dependencies
+++++++++++++++++++++
+
+#. Clone python-docs-samples and change directory to the sample directory you want to use.
+
+    .. code-block:: bash
+
+        $ git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
```

Comment:
Please change `python-docs-samples` to `python-iot`.

Reply:
same as https://github.com/googleapis/python-iot/pull/194#discussion_r715474402